devsetup/Makefile

```diff
@@ -84,6 +84,18 @@ crc_attach_default_interface_cleanup: ## Detach default libvirt network from CRC
 	virsh --connect=qemu:///system detach-interface crc network --mac "$$MAC_ADDRESS"
 	-virsh --connect=qemu:///system net-update default delete ip-dhcp-host "<host name='crc'/>" --config --live
 	sleep 5
```

Comment:
.PHONY missing here compared to other targets, just want to confirm if thats intentional.

Reply:
ack